packages/manager/apps/zimbra/public/translations/domains/diagnostic/Messages_fr_FR.json

```diff
@@ -0,0 +1,35 @@
+  "zimbra_domain_modal_diagnostic_srv_action_validate": "Valider",
+  "zimbra_domain_modal_diagnostic_srv_action_cancel": "Annuler",
+  "zimbra_domain_modal_diagnostic_srv_action_close": "Fermer",
+  "zimbra_domain_modal_diagnostic_srv_field_subdomain": "Sous domaine : <strong>{{subdomain}}</strong>",
+  "zimbra_domain_modal_diagnostic_srv_field_priority": "Priorité : <strong>{{priority}}</strong>",
+  "zimbra_domain_modal_diagnostic_srv_field_weight": "Poids : <strong>{{weight}}</strong>",
+  "zimbra_domain_modal_diagnostic_srv_field_port": "Port : <strong>{{port}}</strong>",
+  "zimbra_domain_modal_diagnostic_srv_field_target": "Cible : <strong>{{target}}</strong>",
+  "zimbra_domain_modal_diagnostic_mx_title": "Diagnostique MX",
+  "zimbra_domain_modal_diagnostic_mx_content_header": "La configuration de l'enregistrement MX permet de recevoir vos emails dans vos boites mail.<br/>Votre domaine n'étant pas géré par OVH Cloud, cette action doit être effectuée manuellement.<br/>Veuillez créer les champs MX sur ce domaine en donnant les mêmes informations que ci-dessous.",
+  "zimbra_domain_modal_diagnostic_mx_content_header_ovh_hosted_domain": "La configuration de l'enregistrement MX permet de recevoir vos emails dans vos boites mail.<br/>Votre domaine étant hébergé chez OVH Cloud, nous allons effectuer la configuration pour vous.<br/>Souhaitez vous commencer à recevoir les mails du domaine <strong>{{domain}}</strong> sur ce service ?",
+  "zimbra_domain_modal_diagnostic_mx_content_footer": "En cas de problème, consultez la documentation de votre fournisseur de domaine.",
+  "zimbra_domain_modal_diagnostic_mx_action_validate": "Confirmer",
+  "zimbra_domain_modal_diagnostic_mx_action_test": "Tester",
+  "zimbra_domain_modal_diagnostic_mx_action_cancel": "Annuler",
+  "zimbra_domain_modal_diagnostic_mx_action_close": "Fermer",
+  "zimbra_domain_modal_diagnostic_mx_fields": "Champs MX",
+  "zimbra_domain_modal_diagnostic_mx_field": "Priorité: <strong>{{priority}}</strong> ; Cible: <strong>{{target}}</strong>",
+  "zimbra_domain_modal_diagnostic_spf_title": "Diagnostique SPF",
+  "zimbra_domain_modal_diagnostic_spf_content_header": "Si le domaine n'est pas géré par le même identifiant client que le service Exchange ou si votre domaine n'est pas hébergé chez OVHcloud, la configuration automatique n'est pas possible. Veuillez créer le champ SPF sur ce domaine manuellement en donnant les mêmes informations que ci-dessous.",
+  "zimbra_domain_modal_diagnostic_spf_content_header_ovh_hosted_domain": "SPF permet de valider qu'OVHCloud est un émetteur légitime de mails avec votre nom de domaine.<br/>Votre domaine étant hébergé chez OVHCloud, nous pouvons procéder à la configuration automatique.<br/>Souhaitez vous effectuer la configuration ?",
+  "zimbra_domain_modal_diagnostic_spf_fields": "Champs SPF",
+  "zimbra_domain_modal_diagnostic_spf_field_subdomain": "Sous domaine : <strong>{{subdomain}}</strong>",
```

Comment:
**nitpick:** dynamic values can be directly displayed in TSX instead of adding it in Translations files.